courses/free-courses-sv.md

```diff
@@ -1,8 +1,14 @@
 ### Index
 
 * [C#](#csharp)
+* [Scratch](#scratch)
 
 
 ### <a id="csharp"></a>C\#
 
 * [ProgSharp](https://www.progsharp.se) - Simon Eddeland
+
+
+### <a id="scratch"></a>Scratch
+
+* [Scratch](http://scratchprogrammering.blogspot.com) - CoderDojo Lund och Daniel Johansson
```

Comment:
how is this a course?

Reply:
Well its a guideline with information and methods how to learn Scratch

Reply:
I understand that the concept behing "Course" could vary betweend different individuals, hence i will remove this PR